### PR TITLE
chore(doc): Update kfp-kubenetes release instructions.

### PR DESCRIPTION
**Pull request by chensun**
**Description of your changes:**

**Checklist:**
- [ ] You have [signed off your commits](https://www.kubeflow.org/docs/about/contributing/#sign-off-your-commits)
- [ ] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention).

### CODE REVIEW

**Review by hbelmiro — APPROVED** [review]
/lgtm
/hold in case it needs anyone else's review.

File: kubernetes_platform/python/RELEASE.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Some steps require elevated permissions to push branches, publish the package, a
 
     **Create and merge the PR into the `master` branch.**
 
-2.  [Requires repo OWNER permissions]  Replace the `KFP_KUBERNETES_VERSION` value with the version in
+1.  [Requires repo OWNER permissions]  Replace the `KFP_KUBERNETES_VERSION` value with the version in
```
Comment (gmfrasca) [inline kubernetes_platform/python/RELEASE.md:12]: nitpick: It looks like the numbered list here and below accidentally updated (each item is Step 1)

Comment (gmfrasca) [reply kubernetes_platform/python/RELEASE.md:12]: ah, auto-increments doing it this way.  TIL.

**Pull request merged**